packages/kbn-cell-actions/.storybook/main.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+module.exports = require('@kbn/storybook').defaultConfig;
```

Comment:
I think we need an alias to execute kbn-cell-actions storybook.

Like this one: https://github.com/elastic/kibana/blob/cd7c331b323d8626875f9646a09d56bc7708d10a/src/dev/storybook/aliases.ts#LL44C5-L44C5